### PR TITLE
chore(IT Wallet): [SIW-1510] Add European Health Insurance Card to the credential list

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the European Health Insurance Card to the credential list in the onboarding screen

## List of changes proposed in this pull request
- Added `CredentialType.EUROPEAN_HEALTH_INSURANCE_CARD` to `availableCredentials`

## How to test
From the wallet screen tap on `Add to wallet` CTA, check that the European Health Insurance Card button is present.

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -3154,6 +3154,10 @@ features:
           issuedDate: "Valida dal"
           expirationDate: "Valida fino"
           restrictionConditions: "Codici"
+      unrecognizedData:
+        title: "Non riconosci alcuni dati?"
+        body: "Se ci sono errori nei dati o vuoi capire meglio cosa significano, puoi contattare il Ministero dell’Interno."
+        cta: "Chiedi informazioni o segnala errori"
```
Comment (LazyAfternoons) [inline locales/en/index.yml:3160]: Is this change intended?

Comment (mastro993) [reply locales/en/index.yml:3160]: Nope, it is a change that comes up after every merge of the base branch 😅

Comment (mastro993) [reply locales/en/index.yml]: Removed here https://github.com/pagopa/io-app/pull/6163/commits/03f76f640138c28953f263dbf02ca3670e88d49f

**Review by LazyAfternoons — APPROVED** [review]
LGTM
